src/components/Modal/index.js

```diff
@@ -67,15 +61,16 @@ const Modal = ({ children, title, isOpen, onClose, ...rest }) => (
           <Heading as="h2">{title}</Heading>
         </Column>
 
-        <CloseButton
+        <Button
           data-testid="closeBtn"
           square
           type="button"
-          as="button"
```

Comment:
add $as

Reply:
Isn't necessary; component already renders a `<button>` element. `$as` Would only be required if I would want to render the `Button` component as a `<span>`, for instance.